.eslintrc

```diff
@@ -17,9 +15,4 @@ globals:
   XPCOMUtils: false
```

Comment:
This isn't like the _most_ awesome thing ever, since we have a "global" for each module / service.

Is it possible it would be cleaner to include comments at the top of each file with the list of modules? That would tend to be close to where the modules are imported, so I might have a marginally better shot at remembering to update this over time.

I'm not sure of the answer here, curious what you think.